src/Stratis.Bitcoin.IntegrationTests/BlockStore/BlockStoreTests.cs

```diff
@@ -181,17 +181,5 @@ public void BlockStoreIndexTx()
                 Assert.Equal(bestBlock1.Transactions.First().GetHash(), trx.GetHash());
             }
         }
-
-        [Fact]
-        public void GetBlockCanRetreiveGenesis()
```

Comment:
This test is already in `BlockRepositoryTests.InitializesGenesisBlockAndTxIndexOnFirstLoad`